src/App.jsx

```diff
@@ -0,0 +1,28 @@
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import HomePage from "@/pages/Home";
+import LogIn from "./pages/LogIn";
+import SignUp from "./pages/LogIn/Components/SignUp";
+import SignIn from "./pages/LogIn/Components/SignIn";
+import "./App.css";
+import Items from "./pages/Items";
+import AddItem from "./pages/AddItem";
+import Product from "./pages/Items/Product";
+
+function App() {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/LogIn" element={<LogIn />} />
+        <Route path="/SignUp" element={<SignUp />} />
+        <Route path="/SignIn" element={<SignIn />} />
+        <Route path="/Items" element={<Items />} />
```

Comment:
❗️ 수정요청
`품 상세 페이지 주소는 "/items/{productId}" 입니다.` 와 같은 요청 사항에 적혀있는 주소로 설정 하시는 것을 추천드려요!

```suggestion
        <Route path="/items" element={<Items />} />
```